.github/workflows/release-web.yml

```
@@ -17,7 +17,26 @@ jobs:
       TEST_ENDPOINT: app.onekeytest.com
     if: ${{ !github.event.workflow_run || (github.event.workflow_run && github.event.workflow_run.conclusion == 'success') }}
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout Source Code
+        uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: Run Shared Env Setup  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)